src/ensembl/src/content/app/browser/drawer/drawer-views/gene-summary/GeneSummary.scss

```diff
@@ -0,0 +1,41 @@
+@import 'src/styles/common';
+
+.standardLabelValue {
+    display: grid;
```

Comment:
2 spaces for indent are mixed with 4 spaces in this file. We use 2.

Reply:
Updated it